Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialFunctions/EvaluateEyeSurface.azsli

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/Features/PBR/Surfaces/EyeSurface.azsli>
+#include <Atom/Features/MatrixUtility.azsli>
+
+void EvaluateEyeSurface(
+    float3 normal,
+    float3 localPosition,
+    float2 uv[UvSetCount],
+    float3 tangents[UvSetCount],
+    float3 bitangents[UvSetCount],
+    bool isFrontFace,
+    inout Surface surface)
+{
+    float3x3 uvMatrix = CreateIdentity3x3(); // By design, only UV0 is allowed to apply transforms. Eye materials are only using UV1 (unwrapped), so no transformation is needed.
```

Comment:
Since Eye.materialtype doesn't even have UV transform properties, you could just remove the comment here, or change to...
```suggestion
    float3x3 uvMatrix = CreateIdentity3x3(); // The eye material type doesn't support UV transforms
```

Reply:
Done.